### PR TITLE
Cleaning up release build action + small PHPCS changes

### PR DESCRIPTION
**Pull request by tommusrhodus**
### Summary

- Removes application files from release build script
- Re-build translation .pot

### CODE REVIEW

File: includes/tag-functions.php [hunk 1]
```diff
@@ -1925,4 +1925,9 @@ public function __construct( $id ) {
 		$data_title
 	);
 }
-add_shortcode( 'tag_likebutton', 'ttgarden_tag_likebutton' );
+add_shortcode( 'tag_likebutton', 'tumblr3_tag_likebutton' );
+
+function tumblr3_tag_postnotesurl(): string {
```
Comment (rtio) [inline includes/tag-functions.php]:
```suggestion
function ttgarden_tag_postnotesurl(): string {
```

File: includes/tag-functions.php [hunk 2]
```diff
@@ -1925,4 +1925,9 @@ public function __construct( $id ) {
 		$data_title
 	);
 }
-add_shortcode( 'tag_likebutton', 'ttgarden_tag_likebutton' );
+add_shortcode( 'tag_likebutton', 'tumblr3_tag_likebutton' );
```
Comment (rtio) [inline includes/tag-functions.php]:
```suggestion
add_shortcode( 'tag_likebutton', 'ttgarden_tag_likebutton' );
```

File: includes/tag-functions.php [hunk 3]
```diff
@@ -1925,4 +1925,9 @@ public function __construct( $id ) {
...
+
+function tumblr3_tag_postnotesurl(): string {
+	return get_comments_link();
+}
+add_shortcode( 'tag_postnotesurl', 'tumblr3_tag_postnotesurl' );
```
Comment (rtio) [inline includes/tag-functions.php]:
```suggestion
add_shortcode( 'tag_postnotesurl', 'ttgarden_tag_postnotesurl' );
```

**Review by rtio — CHANGES_REQUESTED** [review]
Some consistency changes, overall code LGTM

File: includes/tag-functions.php (hunk 1 above)
Comment (tommusrhodus) [reply includes/tag-functions.php]: 🤦‍♂️

Comment (rtio) [reply includes/tag-functions.php]: interesting that phpcs didn't catch this

**Pull request merged**